packages/api/lib/writeRecords/write-granules.js

```diff
@@ -384,102 +281,183 @@ const _generateFilesFromGranule = async ({
 };
 
 /**
- * Write a granule to DynamoDB and PostgreSQL
- *
- * @param {Object} params
- * @param {string} params.collectionId - Collection ID for the workflow
- * @param {Object} params.granule - An API granule object
- * @param {Object} params.provider - Provider object
- * @param {string} params.workflowStatus - Workflow status
- * @param {string} params.collectionCumulusId
- *   Cumulus ID for collection referenced in workflow message, if any
- * @param {string} params.executionCumulusId
- *   Cumulus ID for execution referenced in workflow message, if any
- * @param {Knex} params.knex - Client to interact with PostgreSQL database
- * @param {Object} [params.error] - Workflow error, if any
- * @param {string} [params.executionUrl]
- *   Step Function execution URL for the workflow, if any
- * @param {Object} [params.processingTimeInfo]
- *   Processing time information for the granule, if any
- * @param {string} [params.providerCumulusId]
- *   Cumulus ID for provider referenced in workflow message, if any
- * @param {string} [params.pdrCumulusId]
- *   Cumulus ID for PDR referenced in workflow message, if any
- * @param {Object} [params.granuleModel]
- *   Optional override for the granule model writing to DynamoDB
- *
- * @returns {Promise}
- * @throws
+ * Write a granule record to DynamoDB and PostgreSQL
+ * param {PostgresGranule} postgresGranuleRecord,
+ * param {DynamoDBGranule} dynamoGranuleRecord,
+ * param {number} collectionCumulusId,
+ * param {number} executionCumulusId,
+ * param {Knex} knex,
+ * param {Object} granuleModel = new Granule(),
+ * returns {Promise}
+ * throws
  */
 const _writeGranule = async ({
-  collectionId,
-  granule,
-  pdrName,
-  provider,
-  workflowStartTime,
-  workflowStatus,
-  queryFields,
+  postgresGranuleRecord,
+  dynamoGranuleRecord,
   collectionCumulusId,
   executionCumulusId,
   knex,
-  error,
-  executionUrl,
-  processingTimeInfo,
-  providerCumulusId,
-  pdrCumulusId,
-  granuleModel,
-  updatedAt = Date.now(),
+  granuleModel = new Granule(),
+  granulePgModel = new GranulePgModel(),
```

Comment:
```suggestion
  granuleModel,
  granulePgModel,
```

The parameters are necessary but the initialization is not (I don't believe) since everything that calls `_writeGranule` has already initialized these models

Reply:
Yeah that's probably just belts and suspenders.  We can add it back for all of these after the fact when we need them.